tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -1691,7 +1691,7 @@ public function testIncrementOnExistingModelCallsQueryAndSetsAttribute()
         $model->publicIncrement('foo', 1, ['category' => 1]);
         $this->assertEquals(4, $model->foo);
         $this->assertEquals(1, $model->category);
-        $this->assertTrue($model->isDirty('category'));
```

Comment:
The test case...

I think it is not correct, when id changed the extra data, it shoud be not dirty.